src/components/registerDelegate/registerDelegate.test.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import chai, { expect } from 'chai';
+import { mount } from 'enzyme';
+import chaiEnzyme from 'chai-enzyme';
+import sinon from 'sinon';
+import Lisk from 'lisk-js';
+import RegisterDelegate from './registerDelegate';
+import * as delegateApi from '../../utils/api/delegate';
+
+chai.use(chaiEnzyme());
+
+const normalAccount = {
+  isDelegate: false,
+  address: '16313739661670634666L',
+  balance: 1000e8,
+};
+
+const delegateAccount = {
+  isDelegate: true,
+  address: '16313739661670634666L',
+  balance: 1000e8,
+  delegate: {
+    username: 'lisk-nano',
+  },
+};
+
+const withSecondSecretAccount = {
+  isDelegate: true,
+  address: '16313739661670634666L',
+  balance: 1000e8,
+  delegate: {
+    username: 'lisk-nano',
+  },
+  secondSecret: 'sample phrase',
+};
+
+const props = {
+  peers: {
+    data: Lisk.api({
+      name: 'Custom Node',
+      custom: true,
+      address: 'http://localhost:4000',
+      testnet: true,
+      nethash: '198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d',
+    }),
+  },
+  closeDialog: () => {},
+  onAccountUpdated: () => {},
+  showSuccessAlert: () => {},
```

Comment:
I noticed that I'm not doing this either (in `send.test.js`). I tried to but the spy does not seem to be called.